test/unit/server/ExpressHttpServer.test.ts

```diff
@@ -0,0 +1,82 @@
+import { ExpressHttpServer } from '../../../src/server/ExpressHttpServer';
+import { HttpHandler } from '../../../src/server/HttpHandler';
+import { HttpRequest } from '../../../src/server/HttpRequest';
+import { HttpResponse } from '../../../src/server/HttpResponse';
+import request from 'supertest';
+import { Server } from 'http';
+
+const handle = async(input: { request: HttpRequest; response: HttpResponse }): Promise<void> => {
+  input.response.writeHead(200);
+  input.response.end();
+};
+
+class SimpleHttpHandler extends HttpHandler {
+  public async canHandle(): Promise<void> {
+    return undefined;
```

Comment:
I've seen this a couple of times now, and always do a double-take.
Should we make this a `Promise<boolean>` (but still make it the contract that it's either `true` or an error)?
Or alternatively, rename this to `ensureSupported` or whatever?

Reply:
I have no strong opinions on this so can go any way for me

Reply:
Is just `return` possible actually? It's the `undefined` that makes me think.

Reply:
The linter then complains about having an unnecessary return statement. And if you completely remove it, it complains about having an empty function. :D

Reply:
I'd suggest `boolean` then; can be a separate PR.

Reply:
For completeness, the previous discussion where this changed to void was [here](https://github.com/solid/community-server/pull/3#discussion_r431813854)